.github/container/Dockerfile.base

```diff
@@ -17,13 +18,40 @@ RUN apt-get update && \
         git \
         lld \
         vim \
+        bat \
+        curl \
+        git \
+        gnupg \
+        rsync \
         python-is-python3 \
         python3-pip \
+        liblzma-dev \
         wget \
     && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
-RUN pip install --upgrade --no-cache-dir pip
+RUN <<"EOF" bash -ex
+git config --global user.name "JAX Toolbox"
+git config --global user.email "jax@nvidia.com"
+EOF
+RUN mkdir -p /opt/pip-tools.d
+ADD --chmod=777 \
+    get-source.sh \
+    pip-finalize.sh \
+    /usr/local/bin/
+RUN wget https://github.com/mikefarah/yq/releases/latest/download/yq_linux_$(dpkg --print-architecture) -O /usr/local/bin/yq && \
+    chmod 777 /usr/local/bin/yq
+RUN git clone -b 23.3.1 https://github.com/pypa/pip.git /opt/pip
+# Patch is specific to 23.3.1
+# Generated via: "git diff > pip-vcs-equivalency.patch"
+ADD pip-vcs-equivalency.patch /opt/pip/
```

Comment:
Did you create an issue for JAX-Toolbox as a reminder to remove this hack when and if the change will be merged to pip?

Reply:
I created an internal nvbug for this